cltk/corpus/sanskrit/corpora.py

```diff
@@ -2,4 +2,8 @@
         {'name':'sanskrit_text_jnu',
         'location':'remote',
         'type':'text'},
+        {'name':'sanskrit_text_dcs',
+        'location':'remote',
+        'type':'text'
+            },
```

Comment:
@InsomniacSabbir you should remove the `,` on 8th line and place the `}` at the end of 7th line.